docs/ADD_NEW_EXCHANGE.md

```diff
@@ -0,0 +1,1079 @@
+// Public calls - wrapper functions
+
+// Fetches current ticker information
+tick, err := e.FetchTicker() // e -> f 
+if err != nil {
+  // Handle error
+}
+
+// Fetches current orderbook information
+ob, err := e.FetchOrderbook() // e -> f (do so for the rest of the functions too)
+if err != nil {
+  // Handle error
+}
+```
+
+- Run documentation.go to generate readme file for the exchange:
+```bash
+cd gocryptotrader\cmd\documentation
+go run documentation.go
+```
+
+This will generate a readme file for the exchange which can be found in the new exchange's folder
+
+### Create functions supported by the exchange:
+
+#### Requester functions:
+
+```go
+// SendHTTPRequest sends an unauthenticated HTTP request
+func (f *FTX) SendHTTPRequest(path string, result interface{}) error {
+	return f.SendPayload(context.Background(), &request.Item{
+		Method:        http.MethodGet,
+		Path:          path,
+		Result:        result,
+		Verbose:       f.Verbose,
+		HTTPDebugging: f.HTTPDebugging,
+		HTTPRecording: f.HTTPRecording,
+	})
+}
+```
+
+#### Unauthenticated Functions:
+
+https://docs.ftx.com/#get-markets
+
+Create a type struct in types.go for the response type shown on the documentation website:
+
+```go
+// MarketData stores market data
+type MarketData struct {
+	Name           string  `json:"name"`
+	BaseCurrency   string  `json:"baseCurrency"`
+	QuoteCurrency  string  `json:"quoteCurrency"`
+	MarketType     string  `json:"type"`
+	Underlying     string  `json:"underlying"`
+	Enabled        bool    `json:"enabled"`
+	Ask            float64 `json:"ask"`
+	Bid            float64 `json:"bid"`
+	Last           float64 `json:"last"`
+	PriceIncrement float64 `json:"priceIncrement"`
+	SizeIncrement  float64 `json:"sizeIncrement"`
+}
+```
+
+Create new consts to define endpoint strings, they are created at the top of ftx.go file:
+```go
+const (
+	ftxAPIURL = "https://ftx.com/api"
+
+	// Public endpoints
+	getMarkets           = "/markets"
+	getMarket            = "/markets/"
+	getOrderbook         = "/markets/%s/orderbook?depth=%s"
+	getTrades            = "/markets/%s/trades?"
+	getHistoricalData    = "/markets/%s/candles?"
+	getFutures           = "/futures"
+	getFuture            = "/futures/"
+	getFutureStats       = "/futures/%s/stats"
+	getFundingRates      = "/funding_rates"
+  	getAllWallegetAllWalletBalances = "/wallet/all_balances"
+  ```
+
+Create a get function in ftx.go file and unmarshall the data in the created type:
+```go
+// GetMarkets gets market data
+func (f *FTX) GetMarkets() (Markets, error) {
+	var resp Markets
+	return resp, f.SendHTTPRequest(ftxAPIURL+getMarkets, &resp)
+}
+```
+
+Create a test function in ftx_test.go to see if the data is received and unmarshalled correctly
+```go
+const(
+	spotPair = "FTT/BTC"
+)
+
+func TestGetMarket(t *testing.T) {
+	t.Parallel() // Adding t.Parralel() is preferred as it allows tests to run simultaneously, speeding up package test time
+	f.Verbose = true // used for more detailed output
```

Comment:
Either capitalise `Used` here and other inline comments, or lowercase `Adding` in the above line. Mostly for consistency